cloudbuild.yaml

```diff
@@ -16,7 +16,7 @@ substitutions:
 steps:
   # TODO: currently gcr.io/k8s-testimages/gcb-docker-gcloud has not moved to Artifact Registry
   # gcr.io will be shut down 18 Mar 2025, and we need replacement before then. Latest info below:
-  # https://github.com/kubernetes/test-infra/blob/master/images/gcb-docker-gcloud/cloudbuild.yaml
```

Comment:
There is still no replacement for `gcr.io/k8s-testimages/gcb-docker-gcloud`. The best I can do now is update our TODO note with latest info and progress.